src/cfnlint/data/ExtendedSpecs/all/08_registry_value_types.json

```diff
@@ -0,0 +1,1883 @@
+[
```

Comment:
thoughts on ditching these files + writing directly to [`CloudSpecs`](https://github.com/aws-cloudformation/cfn-python-lint/blame/master/src/cfnlint/data/CloudSpecs) if we make this part of `cfn-lint --update-specs`?

think it's confusing that [`ExtendedSpecs/all`](https://github.com/aws-cloudformation/cfn-python-lint/tree/master/src/cfnlint/data/ExtendedSpecs/all) is partially manually maintained and partially autogenerated

might be useful to include some metadata that the information is coming from the Registry though

Reply:
agreed.  The automation portions can get confusing.  Let me see what I can do.

Reply:
Current values that would need to get corrected in `AWS::DataBrew::Job.DatasetName`
```
DatasetName" : {
      "description" : "Dataset name",
      "type" : "string",
      "pattern" : "[ -퟿-�𐀀-\t]*",
      "minLength" : 1,
      "maxLength" : 255
    },
```

Reply:
found all [non-ASCII characters](https://stackoverflow.com/questions/3001177/how-do-i-grep-for-all-non-ascii-characters) in [resource schemas](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-type-schemas.html):
```bash
$ brew install coreutils
$ ggrep --color='auto' -P -n "[^\x00-\x7F]" *
aws-appflow-connectorprofile.json:158:          "description" : "A unique alphanumeric identiﬁer used to authenticate a user, developer, or calling program to your API.",
aws-appflow-connectorprofile.json:171:          "description" : "A unique alphanumeric identiﬁer used to authenticate a user, developer, or calling program to your API.",
aws-appflow-connectorprofile.json:175:          "description" : "Application keys, in conjunction with your API key, give you full access to Datadog’s programmatic API. Application keys are associated with the user account that created them. The application key is used to log all requests made to the API.",
aws-appflow-connectorprofile.json:215:          "description" : "The identiﬁer for the desired client.",
aws-appflow-connectorprofile.json:245:          "description" : "The identiﬁer for the user.",
aws-appflow-connectorprofile.json:273:          "description" : "The identiﬁer for the desired client.",
aws-appflow-connectorprofile.json:327:          "description" : "The object key for the destination bucket in which Amazon AppFlow will place the ﬁles.",
aws-appflow-connectorprofile.json:398:          "description" : "A unique alphanumeric identiﬁer used to authenticate a user, developer, or calling program to your API.",
aws-appflow-connectorprofile.json:408:          "description" : "The identiﬁer for the desired client.",
aws-appflow-connectorprofile.json:454:          "description" : "The name of the Snowﬂake warehouse.",
aws-appflow-connectorprofile.json:458:          "description" : "The name of the Amazon S3 stage that was created while setting up an Amazon S3 stage in the\nSnowﬂake account. This is written in the following format: < Database>< Schema><Stage Name>.",
aws-appflow-connectorprofile.json:462:          "description" : "The name of the Amazon S3 bucket associated with Snowﬂake.",
aws-appflow-connectorprofile.json:466:          "description" : "The bucket prefix that refers to the Amazon S3 bucket associated with Snowﬂake.",
aws-appflow-connectorprofile.json:470:          "description" : "The Snowﬂake Private Link service name to be used for private data transfers.",
aws-appflow-connectorprofile.json:478:          "description" : "The region of the Snowﬂake account.",
aws-appflow-connectorprofile.json:522:          "description" : "The identiﬁer for the desired client.",
aws-databrew-job.json:9:      "pattern" : "[ -퟿-�𐀀-􏿿\t]*",
aws-databrew-job.json:27:      "pattern" : "[ -퟿-�𐀀-􏿿\t]*",
aws-databrew-job.json:72:      "pattern" : "[ -퟿-�𐀀-􏿿\t]*",
aws-databrew-recipe.json:8:      "pattern" : "[ -퟿-�𐀀-􏿿\t]*",
aws-databrew-recipe.json:16:      "pattern" : "^[ -.0-^`-퟿-�]*$",
aws-databrew-schedule.json:37:      "pattern" : "[ -퟿-�𐀀-􏿿\t]*",
aws-gamelift-gameservergroup.json:244:      "pattern" : "[ -퟿-�𐀀-􏿿\r\n\t]*",
aws-kinesis-stream.json:26:      "description" : "An arbitrary set of tags (key–value pairs) to associate with the Kinesis stream.",
aws-kinesis-stream.json:73:      "description" : "An arbitrary set of tags (key–value pairs) to associate with the Kinesis stream.",
aws-mediapackage-packagingconfiguration.json:100:          "description" : "This setting controls how ad markers are included in the packaged OriginEndpoint. “NONE” will omit all SCTE-35 ad markers from the output. “PASSTHROUGH” causes the manifest to contain a copy of the SCTE-35 ad markers (comments) taken directly from the input HTTP Live Streaming (HLS) manifest. “SCTE35_ENHANCED” generates ad markers and blackout tags based on SCTE-35 messages in the input source.",
aws-mediapackage-packagingconfiguration.json:141:          "description" : "The Dynamic Adaptive Streaming over HTTP (DASH) profile type. When set to “HBBTV_1_5”, HbbTV 1.5 compliant output is enabled.",
aws-mediapackage-packagingconfiguration.json:217:          "description" : "A list of triggers that controls when the outgoing Dynamic Adaptive Streaming over HTTP (DASH) Media Presentation Description (MPD) will be partitioned into multiple periods. If empty, the content will not be partitioned into more than one period. If the list contains “ADS”, new periods will be created where the Asset contains SCTE-35 ad markers.",
```
https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-appflow/pull/6
https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-databrew/pull/2
https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-gamelift/pull/13
https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-kinesis/pull/6
https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-mediapackage/pull/3
(also added to the TODO list in https://github.com/aws-cloudformation/cloudformation-cli/pull/663)